obal/data/roles/setup_sources/tasks/main.yml

```diff
@@ -5,28 +5,14 @@
   tags:
     - always
 
-- name: get spec file
-  find:
-    pattern: "*.spec"
-    path: "{{ package_dir }}"
-  register: spec_file
+- name: 'Setup sources from specfile'
+  include_tasks: specfile.yml
+  when: git is not defined
```

Comment:
Should this be `setup_sources_git` ?

Reply:
Yup, relic from testing with pulp's current nightly code.  fixed